### PR TITLE
Add first draft of stock and flow schema and example

### PR DESCRIPTION
**Pull request by bgyori**
This is a proposed stock and flow AMR which reuses as many components as possible from existing AMRs while following the overall structure of the stock and flow ACSet.

### CODE REVIEW

**Comment by mattprintz** [comment]
Question: How do we represent a flow that upstreams or downstreams from/to a "cloud" rather than a defined stock?

**Comment by bgyori** [comment]
> Question: How do we represent a flow that upstreams or downstreams from/to a "cloud" rather than a defined stock?

I think the placeholder for the "cloud" would just be left empty, i.e.,  `upsteram_stock: null`.

File: stockflow/stockflow_schema.json [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+          },
+          "properties": {
+            "$ref": "#/$defs/properties"
+          }
+        },
+        "required": ["id", "upstream_stock", "downstream_stock"]
+      }
+    },
+    "links": {
+      "type": "array",
```
Comment (mattprintz) [inline stockflow/stockflow_schema.json:131]: I can foresee us needed to include additional information on the links, such as "color"/"category", "polarity", "label", etc.
I am not sure which, if any, of these should be included in the schema or if we should just include a `meta`/`extras` property to store any of this extra information.

**Comment by enoriega** [comment]
@brandomr @bgyori How stable is the amr's schema for stock and flow models?
What particular elements of such AMR are candidates for linking with extractions? I am looking at:
- flows
- stocks
- auxiliaries
- ode parameters

Does it make sense to link those classes of elements?

File: stockflow/stockflow_schema.json (hunk 1 above)
Comment (bgyori) [reply stockflow/stockflow_schema.json:131]: I would put any of those keys under an optional `properties` block.

**Comment by bgyori** [comment]
> @brandomr @bgyori How stable is the amr's schema for stock and flow models? What particular elements of such AMR are candidates for linking with extractions? I am looking at:
> 
> * flows
> * stocks
> * auxiliaries
> * ode parameters
> 
> Does it make sense to link those classes of elements?

That sounds right. In terms of the schema: there haven't been any further comments since I posted this 2 weeks ago to request changes so I think we'll go with what we have here.

File: stockflow/examples/sir.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "header": {
     "name": "SIR Model",
-    "schema": "",
+    "schema": "https://raw.githubusercontent.com/DARPA-ASKEM/Model-Representations/stock-flow/stockflow/stockflow_schema.json",
```
Comment (bgyori) [inline stockflow/examples/sir.json:4]: This is a good temporary addition, thanks! We will ideally change this to a tagged version URL once the PR is merged.

Comment (liunelson) [reply stockflow/examples/sir.json:4]: @mwdchang pointed this out so I made this change so we pass all the checks for merger!

**Pull request merged**